G1ANT.Addon.Selenium/G1ANT.Addon.Selenium/SeleniumAddon.cs

```diff
@@ -56,7 +52,7 @@ private void UnpackDrivers()
                         stream.Write(exe.Value, 0, exe.Value.Length);
                     }
                 }
-                catch { }                
+                catch (Exception ex) { RobotMessageBox.Show(ex.Message); }
```

Comment:
👍 
Consider rethrow of the exception.

Reply:
Can't re-throw at this point, if we get exception at 1st file than other files are not going to be extracted at all.